openghg_inversions/hbmcmc/config/openghg_hbmcmc_input_template_example.ini

```diff
@@ -1,187 +1,288 @@
 ; =======================================================================================
 ; Regional Hierarchical Inverse Modelling Environment Example Configuration File
 ; =======================================================================================
-; RHIME example configuration file for CH4. Please check through variable paths and 
-; names before running.
-; Required inputs are marked as such.
+; RHIME-with-OpenGHG example configuration file for CH4 
```

Comment:
If this file is the sample as the template file, but has sensible defaults, then I think maybe we should just make this the template. Having empty values is a bit confusing because the template isn't actually valid, and the values will be replaced when `run_hbmcmc.py` makes a copy.

We should double check this... but as long as the only CH4 specific info is in "required" categories, then that info won't be propagated through to the output from `run_hbmcmc.py`.

(Also, matching the function defaults will help make the output "correct".)